client/package.json

```diff
@@ -31,6 +31,7 @@
     "deploy-hosting": "npm run predeploy && firebase deploy --only hosting"
   },
   "dependencies": {
+    "@dooboo-ui/snackbar": "^0.0.1",
```

Comment:
Let's remove this after changing the code to use `dooboo-ui`.

Reply:
👍